features.org

```diff
@@ -243,68 +272,72 @@ on remote machines with [[https://www.emacswiki.org/emacs/TrampMode][Tramp]], an
 In Org-mode, extensibility isn't an afterthought --- it's baked right into the
 foundations.
 
-* Time Tracking (Clocking)
+* Actionable agendas
   :PROPERTIES:
-  :CUSTOM_ID: clocking
+  :CUSTOM_ID: agendas
   :HTML_CONTAINER_CLASS: side-fig
   :END:
 
-#+attr_html: :class main-image :alt Using Org to keep track of how long various subtasks took
-[[file:resources/img/features/clocking.svg]]
+#+attr_html: :class main-image :alt An example Org agenda, displaying various todo keywords, priorities, categorisation, and scheduling
+[[file:resources/img/features/agenda.svg]]
 
 ** Description
 
-Keep track of the time you spend on your tasks with Org /Clocking/.
+Plan your day, your way, with ease.
 
-Org provides easy keyboard shortcuts to /clock in/ and /clock out/ of tasks.
+With /agenda views/ --- reports generated from a set of tasks --- you
```

Comment:
Minor tweak, I'd change ` --- reports generated from a set of tasks --- ` to ` ---reports generated from a set of tasks--- `.

Reply:
Ah, I see that I am not up on my em dash punctuation rules.